src/librustc_metadata/rmeta/decoder.rs

```diff
@@ -1843,36 +1930,6 @@ impl CrateMetadata {
 
         None
     }
-
-    #[inline]
-    fn def_path_hash(&self, index: DefIndex) -> DefPathHash {
```

Comment:
Could you keep all the changed code in its old locations?
It's hard to read the diff if code is both changed and moved at the same time.

Reply:
This is not implemented on `CrateMetadata` instead of `CrateRoot`, so it needs to be moved.